packages/client/src/components/FilteringArea/FilteringArea.css

```diff
@@ -58,6 +58,7 @@
 .show-all-button {
   width: 100%;
   line-height: 24px;
+  border-color: rgb(140, 92, 2);
 }
 
 .filtering-checkbox {
```

Comment:
I cannot add a comment for the previous code so left it here:
- in line 71 you need to change HEX color code to RGB.
- please, add spaces between CSS stylings like it was before the line 70

Reply:
☑️